api/src/controllers/utils.ts

```diff
@@ -154,8 +155,10 @@ router.post(
 			schema: req.schema,
 		});
 
+		const sanitizedQuery = sanitizeQuery(req.body.query, req.accountability ?? null);
```

Comment:
Great idea to use sanitizeQuery here compared to the previous `if` condition check!